v2/api/insights/v1api20220615/structure.txt

```diff
@@ -217,60 +217,220 @@ github.com/Azure/azure-service-operator/v2/api/insights/v1api20220615
 │   │       └── "User"
 │   ├── Tags: map[string]string
 │   └── Type: *string
-└── ScheduledQueryRule_Spec_ARM: Object (5 properties)
-    ├── Kind: *Enum (2 values)
-    │   ├── "LogAlert"
-    │   └── "LogToMetric"
+├── ScheduledQueryRule_Spec_ARM: Object (5 properties)
+│   ├── Kind: *Enum (2 values)
+│   │   ├── "LogAlert"
+│   │   └── "LogToMetric"
+│   ├── Location: *string
+│   ├── Name: string
+│   ├── Properties: *Object (15 properties)
+│   │   ├── Actions: *Object (2 properties)
+│   │   │   ├── ActionGroups: string[]
+│   │   │   └── CustomProperties: map[string]string
+│   │   ├── AutoMitigate: *bool
+│   │   ├── CheckWorkspaceAlertsStorageConfigured: *bool
+│   │   ├── Criteria: *Object (1 property)
+│   │   │   └── AllOf: Object (9 properties)[]
+│   │   │       ├── Dimensions: Object (3 properties)[]
+│   │   │       │   ├── Name: *string
+│   │   │       │   ├── Operator: *Enum (2 values)
+│   │   │       │   │   ├── "Exclude"
+│   │   │       │   │   └── "Include"
+│   │   │       │   └── Values: string[]
+│   │   │       ├── FailingPeriods: *Object (2 properties)
+│   │   │       │   ├── MinFailingPeriodsToAlert: *int
+│   │   │       │   └── NumberOfEvaluationPeriods: *int
+│   │   │       ├── MetricMeasureColumn: *string
+│   │   │       ├── MetricName: *string
+│   │   │       ├── Operator: *Enum (5 values)
+│   │   │       │   ├── "Equals"
+│   │   │       │   ├── "GreaterThan"
+│   │   │       │   ├── "GreaterThanOrEqual"
+│   │   │       │   ├── "LessThan"
+│   │   │       │   └── "LessThanOrEqual"
+│   │   │       ├── Query: *string
+│   │   │       ├── ResourceIdColumn: *string
+│   │   │       ├── Threshold: *float64
+│   │   │       └── TimeAggregation: *Enum (5 values)
+│   │   │           ├── "Average"
+│   │   │           ├── "Count"
+│   │   │           ├── "Maximum"
+│   │   │           ├── "Minimum"
+│   │   │           └── "Total"
+│   │   ├── Description: *string
+│   │   ├── DisplayName: *string
+│   │   ├── Enabled: *bool
+│   │   ├── EvaluationFrequency: *string
+│   │   ├── MuteActionsDuration: *string
+│   │   ├── OverrideQueryTimeRange: *string
+│   │   ├── Scopes: string[]
+│   │   ├── Severity: *Enum (5 values)
+│   │   │   ├── 0
+│   │   │   ├── 1
+│   │   │   ├── 2
+│   │   │   ├── 3
+│   │   │   └── 4
+│   │   ├── SkipQueryValidation: *bool
+│   │   ├── TargetResourceTypes: string[]
+│   │   └── WindowSize: *string
+│   └── Tags: map[string]string
+├── Webtest: Resource
+│   ├── Owner: resources/v1apiv20191001.ResourceGroup
+│   ├── Spec: Object (16 properties)
+│   │   ├── AzureName: string
+│   │   ├── Configuration: *Object (1 property)
+│   │   │   └── WebTest: *string
+│   │   ├── Description: *string
+│   │   ├── Enabled: *bool
+│   │   ├── Frequency: *int
+│   │   ├── Kind: *Enum (3 values)
```

Comment:
The old API had a `basic` entry here... not sure what we need to do with that (if anything), but it won't roundtrip forward/backwards.

Reply:
Related to https://github.com/Azure/azure-service-operator/issues/3864